packages/browser/src/Component/Widget/SlideShow/SlideShow.js

```diff
@@ -593,7 +593,7 @@ export class SlideShow {
    */
   initInputListener() {
     this.hidePlaneListener = (event) => {
-      if (event.key != 'h') return;
+      if (event.key.toLowerCase() != 'h') return;
```

Comment:
This may be a dumb question, but since we have an input manager (and a PR about case sensitivity), should we not handle case sensitivity here also?

Reply:
Originally slideshow was working with the InputManager, but since our widgets are meant to be contributed to the itowns project we have to make them not dependant of other part of our code (I doubt that the InputManager will be part of the itowns project).